README.org

```diff
@@ -319,18 +243,16 @@ buffer) beyond the current cursor position.
 
 * Query Language
 
-  The summary window can shows  results filtered by criteria specified
-  with a very simple query language, the basis syntax for that language
-  is shown below:
+The summary window can shows results filtered by criteria specified with a very simple query language, the basis syntax for that language is shown below:
 
 #+BEGIN_SRC text
  [file-mask] [(and | or) [not] regex-note [(and | or) [not] regexp-note ...]]
 #+END_SRC
 
-where
+where:
 
   - file-mask :: is a regular expression that should match the path of file
-    the annotation refers to;
+the annotation refers to;
```

Comment:
This change seems to break the markup, at least in Github's little preview.

Reply:
Thanks @bastibe !